civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5356,6 +5356,8 @@ function _getManagerContacts($contact_id) {
  * @return boolean
  */
 function relationActiveAndCurrent($relation){
-  return $relation['is_active'] == 1 && (strtotime($relation['end_date']) >= time() || empty($relation['end_date']))
+  $endDate = empty($relation['end_date']) ? $relation['end_date'] : $relation['end_date'] . " 23:59:59";
```

Comment:
$endDate = empty($relation['end_date']) ? $relation['end_date'] : $relation['end_date'] . " 23:59:59"; 
can we change it to
$endDate = empty($relation['end_date']) ? null : $relation['end_date'] . " 23:59:59"; 
?
I think it's more readable this way